drivers/sensor/bh1749/Kconfig

```diff
@@ -0,0 +1,23 @@
+#
+#  Copyright (c) 2019 Nordic Semiconductor ASA
+#
+#  SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig BH1749
+	bool "BH1749 Sensor"
+	depends on I2C && HAS_DTS_I2C && HAS_DTS_GPIO && SENSOR
+	help
+	  Enable driver for BH1749 sensors.
+
+if BH1749
+
+config BH1749_TRIGGER
+	bool "Trigger for BH1749"
+	depends on GPIO
+	select BH1749_TRIGGER_GLOBAL_THREAD
+
+config BH1749_TRIGGER_GLOBAL_THREAD
```

Comment:
Where is it intended to be used?

Reply:
It is implicitly used as it is the only implemented way of doing interrupt triggering. Do you think I should remove it for now? It could wrap this https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/862/commits/434356f777bcea4c52f4c5069ea37498c21ea4aa#diff-61dfa1b01eae106d9b7c0b025d4c666eR29

Reply:
Yes, I think it will be better to remove it, to avoid confusion (suggesting that selecting this option or not has some influence on the code).

Reply:
Removed